test/common/http/utility_fuzz_test.cc

```diff
@@ -17,17 +17,18 @@ DEFINE_PROTO_FUZZER(const test::common::http::UtilityTestCase& input) {
   }
   case test::common::http::UtilityTestCase::kParseCookieValue: {
     const auto& parse_cookie_value = input.parse_cookie_value();
-    Http::TestHeaderMapImpl headers;
+    Http::HeaderMapImpl headers;
```

Comment:
Why can't we use the Test version that has more assertions? Too slow?

Reply:
Yep, the `addCopy`s are O(n^2) with the assertions and the number of cookies for this testcase is unbounded. I think it's fine because the code is fuzzing the `Http::Utility` class and not HeaderMap, and I think for the sake of testing `parseCookieValue` we shouldn't bound the number of cookie values to fall under the HeaderMap time limit (just to pinpoint if `parseCookieValue` is time consuming).

Reply:
Perfect. Add that as a comment?